taf/repository_tool.py

```diff
@@ -42,6 +43,14 @@
 DISABLE_KEYS_CACHING = False
 
 
+def get_role_metadata_path(role):
```

Comment:
The difference between `targets_path ` and `metadata_path` and these two functions is that the first pair return full path (`repo_path` + metadata/targets dir), while the second pair is independent of `repo_path` - they return paths of files relative to the repository's root. We need this when loading files using git. I think that we should keep the two properties in any case. I moved `get_role_metadata_path` and `get_target_path` so that we wouldn't have to import `METADATA_DIRECTORY_NAME` and `TARGETS_DIRECTORY_NAME` in other modules as well. We could turn them into static methods, but I've been avoiding that since reading some posts about that being a bad practice in Python